src/test/java/com/epam/tc/hw3/injectionDriver/DriverInjector.java

```diff
@@ -0,0 +1,6 @@
+package com.epam.tc.hw3.injectionDriver;
+
+public interface DriverInjector {
```

Comment:
What is the purpose of this interfaces and classes in injectionDriver folder? I don't see any reasons to implement that complicated code